src/features/UserForm/ui/UncontrolledForm.tsx

```diff
@@ -0,0 +1,289 @@
+import { useDispatch } from 'react-redux';
+import { useState, type FormEvent, useRef } from 'react';
+
+import { addEntry } from '../model/formsSlice';
+
+import {
+  AppButton,
+  FormCheckbox,
+  FormInput,
+  RadioGroup,
+  AutocompleteInput,
+  FileInput,
+  PasswordInput,
+} from '@/shared/ui';
+import { passwordChecks, fileToBase64 } from '@/shared/utils';
+
+type UncontrolledFormProps = { onSuccess?: () => void };
+
+export const UncontrolledForm = ({ onSuccess }: UncontrolledFormProps) => {
+  const dispatch = useDispatch();
+
+  const [nameError, setNameError] = useState<string | null>(null);
+  const [ageError, setAgeError] = useState<string | null>(null);
+  const [emailError, setEmailError] = useState<string | null>(null);
+  const [genderError, setGenderError] = useState<string | null>(null);
+  const [countryError, setCountryError] = useState<string | null>(null);
+  const [passwordError, setPasswordError] = useState<string | null>(null);
+  const [confirmPasswordError, setConfirmPasswordError] = useState<
+    string | null
+  >(null);
+  const [pictureError, setPictureError] = useState<string | null>(null);
+  const [termsError, setTermsError] = useState<string | null>(null);
```

Comment:
from my point of view it's better to store all errors in one object, one state. It will reduce boilerplate. And update state using setErrers(prevState=>({...prevState, name: '..'}))